muirwik-components/src/main/kotlin/com/ccfraser/muirwik/components/ThemeProvider.kt

```diff
@@ -48,3 +48,7 @@ fun RBuilder.mThemeProvider(theme: Theme = createMuiTheme(), handler: RHandler<R
     if (handler != null) handler()
 }
 
+fun useTheme(): Theme {
+    return useContext(themeContext)
+}
+
```

Comment:
The Material-UI useTheme does a few other things, not sure if they are important, but just in case, maybe it should be like this instead:

```
@JsModule("@material-ui/core/styles/useTheme")
private external val useThemeDefault: dynamic
fun useTheme() = useThemeDefault.default()
```

Reply:
Why useTheme return dynamic instead of Theme

Reply:
Ops, yes that should be `useTheme(): Theme` and to stop warnings...
```
@Suppress("UNCHECKED_CAST_TO_EXTERNAL_INTERFACE")
fun useTheme(): Theme = useThemeDefault.default() as Theme
```